petab/core.py

```diff
@@ -192,3 +216,31 @@ def flatten_timepoint_specific_output_overrides(
             sbml_model=petab_problem.sbml_model,
             observable_id=replicate_id,
             formula='noiseParameter1_' + replicate_id)
+
+
+def concat_tables(
+        tables: Iterable[Union[pd.DataFrame, str]],
+        parser: Optional[Callable] = None
```

Comment:
file_parser? Then the purpose is clear. I thought the overall table is parsed to sth.